containers/deploy/candlepin/deployments.yml

```diff
@@ -1,27 +1,38 @@
 ---
 - name: Scale Candlepin to 0
-  openshift_v1_deployment_config:
+  openshift_raw:
       name: candlepin
       namespace: "{{ project_name }}"
       replicas: 0
+      kind: DeploymentConfig
   tags:
     - stop
     - restart
 
 - name: Create Candlepin deployment
-  openshift_v1_deployment_config:
+  openshift_raw:
       state: "{{ deployment_state }}"
       force: false
-      resource_definition:
+      definition:
           apiVersion: v1
-          kind: deployment_config
+          kind: DeploymentConfig
           metadata:
               name: candlepin
               labels:
                   app: foreman
                   service: candlepin
               namespace: "{{ project_name }}"
           spec:
+              replicas: 1
+              strategy:
+                resources: {}
+                rollingParams:
+                  intervalSeconds: 1
+                  maxSurge: 25%
+                  maxUnavailable: 25%
+                  timeoutSeconds: 900
+                  updatePeriodSeconds: 1
```

Comment:
Are these things we need because of the new library or separate enhancements?

Reply:
They were me testing something and copy from Openshift itself. I can drop them.